### PR TITLE
Allow separate models for training on poisoned and cleaned datasets

### LINKED ISSUE #582: Poisoning scenario - separate models for training on poisoned and cleaned datasets
Separate training configurations for training on the poisoned dataset versus the cleaned dataset (number of epochs, learning rate, even models)

### PR DESCRIPTION
**Pull request by ng390**
Closes #582, with concurrent PR 46 in armory-examples.

### CODE REVIEW

**Comment by ng390** [comment]
Definitely will do on the documentation. The example config is PR #46 in armory-examples -- can link to it in the documentation once it is merged in.
